[2024-11-12T20:49:26.910+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-12T20:49:26.963+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_commands_dag.dbt_test scheduled__2024-11-11T11:30:00+00:00 [queued]>
[2024-11-12T20:49:26.981+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_commands_dag.dbt_test scheduled__2024-11-11T11:30:00+00:00 [queued]>
[2024-11-12T20:49:26.983+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-11-12T20:49:27.006+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_test> on 2024-11-11 11:30:00+00:00
[2024-11-12T20:49:27.013+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_commands_dag', 'dbt_test', 'scheduled__2024-11-11T11:30:00+00:00', '--job-id', '199', '--raw', '--subdir', 'DAGS_FOLDER/DBT_DAG_RUN.py', '--cfg-path', '/tmp/tmpt_j39tsh']
[2024-11-12T20:49:27.015+0000] {standard_task_runner.py:91} INFO - Job 199: Subtask dbt_test
[2024-11-12T20:49:27.018+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=394) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-12T20:49:27.020+0000] {standard_task_runner.py:63} INFO - Started process 395 to run task
[2024-11-12T20:49:27.081+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_commands_dag.dbt_test scheduled__2024-11-11T11:30:00+00:00 [running]> on host 16d7368eac06
[2024-11-12T20:49:27.276+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Utkarsh Tripathi' AIRFLOW_CTX_DAG_ID='dbt_commands_dag' AIRFLOW_CTX_TASK_ID='dbt_test' AIRFLOW_CTX_EXECUTION_DATE='2024-11-11T11:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-11T11:30:00+00:00'
[2024-11-12T20:49:27.277+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-12T20:49:27.294+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-12T20:49:27.297+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt test --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt']
[2024-11-12T20:49:27.307+0000] {subprocess.py:86} INFO - Output:
[2024-11-12T20:49:30.011+0000] {subprocess.py:93} INFO - [0m20:49:30  Running with dbt=1.8.7
[2024-11-12T20:49:31.491+0000] {subprocess.py:93} INFO - [0m20:49:31  Registered adapter: snowflake=1.8.4
[2024-11-12T20:49:32.029+0000] {subprocess.py:93} INFO - [0m20:49:32  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2024-11-12T20:49:32.031+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2024-11-12T20:49:32.031+0000] {subprocess.py:93} INFO - - models.build_utl.example
[2024-11-12T20:49:32.358+0000] {subprocess.py:93} INFO - [0m20:49:32  Found 2 models, 1 snapshot, 2 data tests, 2 sources, 459 macros
[2024-11-12T20:49:32.367+0000] {subprocess.py:93} INFO - [0m20:49:32
[2024-11-12T20:49:36.794+0000] {subprocess.py:93} INFO - [0m20:49:36  Concurrency: 1 threads (target='LAB2_WORK')
[2024-11-12T20:49:36.797+0000] {subprocess.py:93} INFO - [0m20:49:36
[2024-11-12T20:49:36.831+0000] {subprocess.py:93} INFO - [0m20:49:36  1 of 2 START test not_null_moving_averages_date_symbol ......................... [RUN]
[2024-11-12T20:49:39.140+0000] {subprocess.py:93} INFO - [0m20:49:39  1 of 2 PASS not_null_moving_averages_date_symbol ............................... [[32mPASS[0m in 2.31s]
[2024-11-12T20:49:39.146+0000] {subprocess.py:93} INFO - [0m20:49:39  2 of 2 START test unique_moving_averages_date_symbol ........................... [RUN]
[2024-11-12T20:49:40.968+0000] {subprocess.py:93} INFO - [0m20:49:40  2 of 2 PASS unique_moving_averages_date_symbol ................................. [[32mPASS[0m in 1.82s]
[2024-11-12T20:49:40.987+0000] {subprocess.py:93} INFO - [0m20:49:40
[2024-11-12T20:49:40.990+0000] {subprocess.py:93} INFO - [0m20:49:40  Finished running 2 data tests in 0 hours 0 minutes and 8.62 seconds (8.62s).
[2024-11-12T20:49:41.104+0000] {subprocess.py:93} INFO - [0m20:49:41
[2024-11-12T20:49:41.107+0000] {subprocess.py:93} INFO - [0m20:49:41  [32mCompleted successfully[0m
[2024-11-12T20:49:41.108+0000] {subprocess.py:93} INFO - [0m20:49:41
[2024-11-12T20:49:41.109+0000] {subprocess.py:93} INFO - [0m20:49:41  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2024-11-12T20:49:42.375+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-12T20:49:42.377+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-12T20:49:42.408+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_commands_dag, task_id=dbt_test, run_id=scheduled__2024-11-11T11:30:00+00:00, execution_date=20241111T113000, start_date=20241112T204926, end_date=20241112T204942
[2024-11-12T20:49:42.440+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-12T20:49:42.467+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-12T20:49:42.473+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
